.rubocop_base.yml

```diff
@@ -197,7 +197,10 @@ Style/MethodCallWithArgsParentheses:
     - die
     - opt
     # MIQ
+    - add_dependency
```

Comment:
These should probably be under `# Ruby` above or a new `# Rubygems` since the methods are not defined in the ManageIQ code base.

Reply:
Good point! Updated